package.json

```diff
@@ -12,7 +12,7 @@
     "build:node": "shx rm -rf lib && tsc -b tsconfig.build.json",
     "build:node:watch": "shx rm -rf lib && tsc -b tsconfig.build.json --watch",
     "prepare": "npm run build:node && npm run build:webpack",
-    "test": "npm run lint && cross-env NODE_ENV=test nyc node ./tests/unitTests/src/index.ts",
+    "test": "npm run lint && cross-env NODE_ENV=test nyc node ./tests/unitTests/src/index.ts | tap-min; nyc report",
```

Comment:
I think the pipe operator `|` here is cross-platform (works on Windows cmd.exe and Powershell), but the `;` operator is definitely not. I think the [npm-run-all](https://www.npmjs.com/package/npm-run-all) tool could be used to perform this same thing in a cross platform way.

Reply:
In general, I think less verbose test output can be useful. I have actually ran into the need to increase my terminal line buffer length to see the full output with large tape test suites. 

However, I don't think it should be the default `test` command. The verbose output is useful for inspecting failed circleCI tests, among other things. 

Can this be made into a separate command, something like `test:min`?